### PR TITLE
Add data model

### PR DESCRIPTION
**Pull request by Aqurds**
The following features were added,

**Created a new app **
**Initialized the project with Git.**
**Makd sure that the project has Postgres database set up.**
**Built the project schema.**
- > Create and run the necessary migration files.
- > Table and column names should match the ERD diagram.
- > photo for users table should be a link to an image
- > Foreign keys should be included.
- > Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
- > All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Aqurds,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly pay attention to your submission requests. This is the data model project but you submitted the validation and model specs form.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/migrate/20230926160040_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :bio
+      t.integer :postCounter
+
+      t.timestamps
+    end
+  end
+end
```
Comment (levy002) [inline db/migrate/20230926160040_create_users.rb]: - [x] Kindly make sure to add the `Photo` column in the `User` table in order to match the project requirement.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Aqurds   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230926160040_create_users.rb (hunk 1 above)
Comment (Aqurds) [reply db/migrate/20230926160040_create_users.rb]: Photo column added

**Review by levy002 — APPROVED** [review]
## Hello @Aqurds 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- You implemented the suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**